AmericanMedicalAssociation/ama-style-guide-2
# EWL-9255: Add alert banner option to category pages in styleguide.

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- N/A

**Jira Ticket**
- [EWL-9255: Alert Banner for Cat/Subcat Pages - Styleguide](https://issues.ama-assn.org/browse/EWL-9255)

## Description
Adds category/subcategory with alert banner to styleguide 


## To Test
- open styleguide
- navigate to pages tab
- select category drop down and open category with alert banner
- confirm styling matches the following zeplins: 
[Desktop ](https://zpl.io/2pqDWmJ)

[Tablet](https://zpl.io/VD358vJ) 

[Mobile](https://zpl.io/adoBnDe )

**NOTE**: The alert banner has new styling that is in a PR but still pending a merge. See here: [https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/1133](https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/1133)

## Visual Regressions
- N/A


## Relevant Screenshots/GIFs
<img width="1095" alt="Screen Shot 2022-01-28 at 2 34 30 PM" src="https://user-images.githubusercontent.com/67962801/151617589-81e24bb2-18eb-466a-bb9f-ac0cb26dc57d.png">

<img width="1095" alt="Screen Shot 2022-01-28 at 2 37 11 PM" src="https://user-images.githubusercontent.com/67962801/151617830-939dc0c6-dc73-409d-8dc3-7abde5e95fd7.png">


## Remaining Tasks
- N/A

## Additional Notes
**NOTE**: The alert banner has new styling that is in a PR but still pending a merge. See here: [https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/1133](https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/1133)

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
